NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/InformationAndConsentViewController.cs

```diff
@@ -10,14 +10,15 @@
 using NDB.Covid19.Utils;
 using UIKit;
 using NDB.Covid19.Enums;
+using NDB.Covid19.ProtoModels;
```

Comment:
same as above, import as static to just refer to `ReportType` in the code so it looks cleaner